Zorgen dat time ago wordt weergegeven bij technical debt end date field (#5787)
ICTU/quality-time: docs/src/changelog.md

- Add the possibility in metrics of type missing metrics to exclude certain subjects from a report. Closes [#5119](https://github.com/ICTU/quality-time/issues/5119).

### Added

> - Add label text to indicate time ago or time to go for date input fields. Closes [#5123](https://github.com/ICTU/quality-time/issues/5123).

Reviewer: ```suggestion
- Add the time ago or time to go to labels of date input fields. Closes [#5123](https://github.com/ICTU/quality-time/issues/5123).
```
